### PR TITLE
Converted msdn links to site relative links in 4 topics

### PR DESCRIPTION
**Pull request by mikeblome**

### CODE REVIEW

File: docs/csharp/classes.md [hunk 1]
```diff
@@ -57,25 +57,25 @@ When a class declares a base class, it inherits all the members of the base clas
...
  
 ## Example
-In the following example, a public class that contains a single field, a method, and a special method called a constructor is defined. For more information, see [Constructors](https://msdn.microsoft.com/library/ace5hbzh.aspx). The class is then instantiated with the **new** keyword.
+In the following example, a public class that contains a single field, a method, and a special method called a constructor is defined. For more information, see [Constructors](programming-guide/classes-and-structs/constructors.md). The class is then instantiated with the **new** keyword.
 
 [!code-csharp[Class Example](../../samples/snippets/csharp/concepts/classes/class-example.cs)]  
   
 ## C# language specification  
-For more information, see the [C# language specification](https://msdn.microsoft.com/library/ms228593.aspx). The language specification is the definitive source for C# syntax and usage.
+For more information, see the [C# language specification](language-reference/language-specification/). The language specification is the definitive source for C# syntax and usage.
```
Comment (mairaw) [inline docs/csharp/classes.md]: missing file name index.md

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @mikeblome. LGTM. We're just getting a couple of warnings.

File: docs/csharp/classes.md (hunk 1 above)
Comment (mikeblome) [reply docs/csharp/classes.md]: @mairaw I fixed all warnings. thanks for heads up.

**Review by mairaw — APPROVED** [review]
LGTM. Let's see if build is green before merging.

**Pull request merged**